libs/ledger-live-common/src/reconciliation.ts

```diff
@@ -307,95 +289,160 @@ export function patchAccount(
     }
   }
 
-  if (
-    updatedRaw.tronResources &&
-    // @ts-expect-error check if this is valid for deep equal check
-    account.tronResources !== updatedRaw.tronResources
-  ) {
-    next.tronResources = fromTronResourcesRaw(updatedRaw.tronResources);
-    changed = true;
-  }
-
-  if (
-    updatedRaw.cosmosResources &&
-    // @ts-expect-error check if this is valid for deep equal check
-    account.cosmosResources !== updatedRaw.cosmosResources
-  ) {
-    next.cosmosResources = fromCosmosResourcesRaw(updatedRaw.cosmosResources);
-    changed = true;
-  }
-
-  if (
-    updatedRaw.algorandResources &&
-    // @ts-expect-error check if this is valid for deep equal check
-    account.algorandResources !== updatedRaw.algorandResources
-  ) {
-    next.algorandResources = fromAlgorandResourcesRaw(
-      updatedRaw.algorandResources
-    );
-    changed = true;
-  }
-
-  if (
-    updatedRaw.bitcoinResources &&
-    shouldRefreshBitcoinResources(updatedRaw, account)
-  ) {
-    next.bitcoinResources = fromBitcoinResourcesRaw(
-      updatedRaw.bitcoinResources
-    );
-    changed = true;
-  }
-
-  if (
-    updatedRaw.polkadotResources &&
-    // @ts-expect-error check if this is valid for deep equal check
-    account.polkadotResources !== updatedRaw.polkadotResources
-  ) {
-    next.polkadotResources = fromPolkadotResourcesRaw(
-      updatedRaw.polkadotResources
-    );
-    changed = true;
-  }
-
-  if (
-    updatedRaw.tezosResources &&
-    account.tezosResources !== updatedRaw.tezosResources
-  ) {
-    next.tezosResources = fromTezosResourcesRaw(updatedRaw.tezosResources);
-    changed = true;
-  }
-
-  if (
-    updatedRaw.elrondResources &&
-    account.elrondResources !== updatedRaw.elrondResources
-  ) {
-    next.elrondResources = fromElrondResourcesRaw(updatedRaw.elrondResources);
-    changed = true;
-  }
-
-  if (
-    updatedRaw.cardanoResources &&
-    // @ts-expect-error check if this is valid for deep equal check
-    account.cardanoResources !== updatedRaw.cardanoResources
-  ) {
-    next.cardanoResources = fromCardanoResourceRaw(updatedRaw.cardanoResources);
-    changed = true;
-  }
-
-  if (
-    updatedRaw.cryptoOrgResources &&
-    // @ts-expect-error types don't overlap ¯\_(ツ)_/¯
-    account.cryptoOrgResources !== updatedRaw.cryptoOrgResources
-  ) {
-    next.cryptoOrgResources = fromCryptoOrgResourcesRaw(
-      updatedRaw.cryptoOrgResources
-    );
-    changed = true;
-  }
+  switch (account.currency.id) {
```

Comment:
(same here as previous comments about family instead of id)

Reply:
Done